framework/core/command_buffer.cpp

```diff
@@ -131,7 +131,11 @@ void CommandBuffer::flush(VkPipelineBindPoint pipeline_bind_point)
 	flush_descriptor_state(pipeline_bind_point);
 }
 
-void CommandBuffer::begin_render_pass(const RenderTarget &render_target, const std::vector<LoadStoreInfo> &load_store_infos, const std::vector<VkClearValue> &clear_values, const std::vector<std::unique_ptr<Subpass>> &subpasses, VkSubpassContents contents)
+void CommandBuffer::begin_render_pass(const RenderTarget                                           &render_target,
+                                      const std::vector<LoadStoreInfo>                             &load_store_infos,
+                                      const std::vector<VkClearValue>                              &clear_values,
+                                      const std::vector<std::unique_ptr<vkb::rendering::SubpassC>> &subpasses,
```

Comment:
FWIW, some compilers will have issues with >> next to each other.  I don't know if it's been fixed in all compilers or not as I just got into a habit of putting a space between the two.  This compiles and works in Linux; and CI passes so this is probably a non-issue.

Reply:
This was resolved with C++11!
I can't imagine that there's a C++11 compiler out in the wild, that does not support those double closing angle brackets.